adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/CoverageRDD.scala

```diff
@@ -251,9 +251,11 @@ case class CoverageRDD(rdd: RDD[Coverage],
    * return a Seq of exactly one ReferenceRegion.
    *
    * @param elem The Coverage to get an underlying region for.
+   * @param stranded True to report stranded data for each Coverage.
    * @return Sequence of ReferenceRegions extracted from Coverage.
    */
-  protected def getReferenceRegions(elem: Coverage): Seq[ReferenceRegion] = {
+  protected def getReferenceRegions(elem: Coverage,
+                                    stranded: Boolean = false): Seq[ReferenceRegion] = {
```

Comment:
We do support generating coverage data for independent/positive/negative strands. I don't know if we expose that anywhere. If we don't, can you open a ticket for us to follow up on this? CC @akmorrow13